docs/pages/guides/analyzing-bundles.mdx

```diff
@@ -10,77 +10,39 @@ Bundle performance varies for different platforms. For example, web browsers don
 
 ## Analyzing bundle size
 
-The libraries used in a project influence the size of the production JavaScript bundle. You can use [source map explorer](https://www.npmjs.com/package/source-map-explorer) to visualize the production bundle and identify which libraries are contributing to the bundle size.
+The libraries used in a project influence the size of the production JavaScript bundle. Starting from **Expo SDK 51**, you can use [Expo Atlas](https://github.com/expo/expo-atlas#readme) to visualize the production bundle and identify which libraries contribute to the bundle size.
 
-<Step label="1">
+### Using Atlas with `expo start`
 
-To use source map explorer, run the following command to install it:
+You can use Expo Atlas with the local development server. This method allows Atlas to update whenever you change any code within your project. 
 
-<Terminal cmd={['$ npm i --save-dev source-map-explorer']} />
+Once your app is up and running on Android, iOS, and/or web, you can open Atlas through the [dev tools plugin menu](/debugging/devtools-plugins/#using-a-dev-tools-plugin) using <kbd>shift</kbd> + <kbd>m</kbd>.
 
-</Step>
+<Terminal cmd={[
+  '# Start the local development server with Atlas',
+  '$ EXPO_UNSTABLE_ATLAS=true npx expo start',
+  '',
+  '# Optionally, run the local development server in production mode',
+  '$ EXPO_UNSTABLE_ATLAS=true npx expo start --no-dev', 
+]} />
 
-<Step label="2">
+> **Changing development mode to production**
+>
+> Expo starts in development mode by default. Development mode may turn off optimizations that are enabled in production mode. You can also start the local development server in production mode, which provides a more accurate representation of the production bundle size.
```

Comment:
```suggestion
#### Changing development mode to production

By default, Expo starts the local development server in [development mode](/workflow/development-mode/#development-mode). Development mode disables some optimizations that are enabled in [production mode](/workflow/development-mode/#production-mode). You can also start the local development server in production mode to get a more accurate representation of the production bundle size:

<Terminal cmd={[
  '# Run the local development server in production mode',
  '$ EXPO_UNSTABLE_ATLAS=true npx expo start --no-dev', 
]} />
```

Let's make production mode info part of the main content by including it as a sub section instead of using a callout and also split the command the since the context in the previous callout was below the callout (not a good reading experience for the user since we're telling them the command first without any context).